settings.py

```diff
@@ -24,7 +24,10 @@
 
 
 def init_sentry(integrations: Sequence[Integration] = ()):
-    if _sentry_dsn:
```

Comment:
a simpler way to disable sentry for dev is to not provide `_sentry_dns` (or having a null value): https://docs.sentry.io/platforms/python/configuration/options/#dsn

Reply:
yep, but our code was raising, that why I put a disabled. At least this is explicit when you don't want sentry, and not that you forgot it.